docs/source/daml/intro/9_Functional101.rst

```diff
@@ -57,10 +57,7 @@ In the case of ``increment`` it could have been omitted. Similarly, we could def
   :start-after: -- ADD_BEGIN
   :end-before: -- ADD_END
 
-If you do this, and wonder what type the compiler has inferred, you can hover over the function name in the IDE:
-
-.. figure:: images/9_Functional101/signature.png
```

Comment:
This image shows something else.

Reply:
Does it? It looks like it’s a screenshot of someone hovering over a function name which seems to be exactly what we want?

Reply:
I expected `add`, and the picture shows `validate`. Or maybe it's too late for me :)

Reply:
Oh I see so it’s a screenshot of hovering over a signature but for the wrong function. Can we replace it by the correct screenshot instead of dropping it completely?

Reply:
I'm not sure if it's very helpful, but sure, we can.

Reply:
Done